Geometry/TrackerGeometryBuilder/test/ModuleInfo.cc

```diff
@@ -226,10 +225,9 @@ ModuleInfo::analyze( const edm::Event& iEvent, const edm::EventSetup& iSetup )
 	std::string name = modules[i]->name().name();
 	if(name == "PixelBarrelActiveFull") pxb_fullN++;
 	if(name == "PixelBarrelActiveHalf") pxb_halfN++;
-	PXBDetId module(rawid);
-	unsigned int theLayer  = module.layer();
-	unsigned int theLadder = module.ladder();
-	unsigned int theModule = module.module();
+	unsigned int theLayer  = tTopo->layer(id);
```

Comment:
same comment as with the other file... I think it's simpler to write `pxbLayer` etc. out